packages/react/package.json

```diff
@@ -17,9 +17,24 @@
     "umd/",
     "jsx-runtime.js",
     "jsx-dev-runtime.js",
+    "unstable-index.server.js",
     "unstable-cache.js"
   ],
   "main": "index.js",
+  "exports": {
+    ".": {
+      "react-server": "./unstable-index.server.js",
```

Comment:
Didn't know this feature existed, that's cool

Reply:
Was thinking the same thing. Just finished reading about it myself.